### ISSUE TITLE
Reproduce on the single GPU

### ISSUE THREAD

**Issue by fufj** [DQiaole/MemFlow#21]
Hi! Thanks for your excellent work.
I got a question when I reproduce this work on a single RTX 4090. The batch_size have to set 1 when train the MemflowNet_T (other seetings keep same.). And I got wosre results than the paper shown.
I wonder whether the GPU and batch_size setting significantly influence the results?

**Comment by DQiaole** [comment]
Hi, I think the batch size will affect the model's performance. You may need to use a larger batch size through gradient accumulation on single GPU.

**Comment by fufj** [comment]
> Hi, I think the batch size will affect the model's performance. You may need to use a larger batch size through gradient accumulation on single GPU.

Thanks for your ans.
But I meet another problem. I wonder know if I neet to pretrain in FlyingChair and FlyingThings3D for 120k (batch size 8) and 150k (batch size 6) iterations when I change the network architecture. Because the pretrained [skflow-things.pth] and [twins_skflow.pth] cannot be used at that situation.

**Comment by DQiaole** [comment]
Sure, you should pretrain the network on 2-frame datasets, which would provide better performance as verified in our paper.

**Comment by fufj** [comment]
> Sure, you should pretrain the network on 2-frame datasets, which would provide better performance as verified in our paper.

Thank you so much. I will try it.
